src/main.c

```diff
@@ -49,6 +52,61 @@ static void softReset_button_callback(struct device *port, struct gpio_callback
 	soft_reboot();
 }
 
+
+void cdc_acm_descriptor_callback (cdc_acm_cfg_t *cfg)
+{
+	customer_data_t *otp_data = (customer_data_t *)CUSTOMER_DATA_ADDRESS;
+
+	memset(cfg->vendor_string, '\0', 32);
```

Comment:
I would prefer the entire storage space for, say, vendor_string, to be cleared out every time when it is being flashed.  This may prevent residuals from old string read from the flash.